pkg/agent/openflow/client.go

```diff
@@ -294,6 +294,17 @@ type Client interface {
 	UninstallTrafficControlReturnPortFlow(returnOFPort uint32) error
 
 	InstallMulticastGroup(ofGroupID binding.GroupIDType, localReceivers []uint32) error
+
+	// InstallVMUplinkFlows installs flows to forward packet between uplinkPort and hostPort. On a VM, the
+	// uplink and host internal port are paired directly, and no layer 2/3 forwarding flows installed.
+	InstallVMUplinkFlows(hostInterfaceName string, hostPort int32, uplinkPort int32) error
+
+	// UninstallHostUplinkFlows removes the flows installed to forward packet between uplinkPort and hostPort.
+	UninstallHostUplinkFlows(hostInterfaceName string) error
+
+	// InstallPolicyBypassFlows installs flows to bypRass the NetworkPolicy rules on the traffic with the given ipnet/ip,
+	// port, proto and direction.
```

Comment:
Could we also add comments to say it is used for particular traffic on a VM to bypass NetworkPolicy enforcement.

Reply:
added.